server/app/services/applicant/ApplicantData.java

```diff
@@ -147,7 +147,11 @@ public boolean updateDidFailAt(Path path) {
   }
 
   public Optional<LocalDate> getDateOfBirth() {
-    return readDate(WellKnownPaths.APPLICANT_DOB);
+    Path dobPath = WellKnownPaths.APPLICANT_DOB;
```

Comment:
This is used to show the DOB on the TI client list page. Since we'll have a mix of both data structures for awhile, we need to handle both cases. I'll remove this when I remove the job.